web/src/test/resources/com/graphhopper/http/resources/truck.yml

```diff
@@ -39,25 +32,30 @@ max_speed_fallback: 100
 distance_influence: 90
 
 # Now we want to set a preference without changing the taken time. The default priority is 1 and does not change something.
-# Higher than 1 will prefer the road under the specified condition and lower than 1 will avoid it.
+# Higher than 1 is not possible directly, only via the catch-all key "*", see road_class where motorways are preferred. Lower than 1 will avoid it.
```

Comment:
Values higher than 1 are not allowed, but instead you can lower the priority of all other entries via the catch-all key "*", see...